core-web/libs/block-editor/src/lib/nodes/contentlet-block/contentlet-block.component.ts

```diff
@@ -7,8 +7,7 @@ import { DotCMSContentlet } from '@dotcms/dotcms-models';
 @Component({
     selector: 'dot-contentlet-block',
     templateUrl: './contentlet-block.component.html',
-    styleUrls: ['./contentlet-block.component.scss'],
-    encapsulation: ViewEncapsulation.None
```

Comment:
why remove ?

Reply:
Is better not to have this `encapsulation: ViewEncapsulation.None` but not sure why was deleted.

Reply:
Since we were using `encapsulation: ViewEncapsulation.None`, the styles in the `contentlet-block.component.scss` were being overwritten [[View Encapsulation Docs](https://angular.io/guide/view-encapsulation)].